test/src/system/st_3drepobouncerClient.cpp

```diff
@@ -123,6 +123,26 @@ static int runProcess(
 #endif
 }
 
+
+static int testUpload (
+	std::string mongoDbName,
+	std::string projectName,
+	std::string fileName
+	)
+{
+	std::string bimUpload = produceUploadArgs(
```

Comment:
`bimUpload` is a bit misleading when you've factored this out to be a generic function

maybe just call it something like `uploadCmd`?

Reply:
its done